amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -606,6 +637,50 @@ describe('SearchPage', () => {
     });
   });
 
+  describe('renderContent', () => {
+    let content;
+    let props;
+    let wrapper;
+    describe('popular tables', () => {
+      beforeAll(() => {
```

Comment:
nit: Since each we only have one case for each test, we don't necessarily need a new `describe` block and a `beforeAll`, we could also have done:

```
it('renders popular tables if searchTerm is empty', () => {
    const setupResult = setup({ searchTerm: '' });
    props = setupResult.props;
    wrapper = setupResult.wrapper;
    expect(wrapper.instance().renderContent()).toEqual(wrapper.instance().renderPopularTables());
});
```
Note the addition of clarifying the conditional logic we're testing in the description as well.